### ISSUE TITLE
iOS: Ver-ID credentials Capture ID card, result not showing

### ISSUE THREAD

**Issue by sanidkpc** [AppliedRecognition/Ver-ID-Credentials-Cordova-Plugin#13]
Hi @jakubdolejs,
I successfully integrated Ver-ID-Credentials-Cordova-Plugin into our app [iOS platform]. 
But got an issues.
Now I can successfully capture a ID card (driving license) [front+back], but ver-id does not show any result.
please observer the code snippet below. Accordingly to the documentation
```js
    let apiSecret = "api Secret"; // Alternative way to set your Ver-ID API secret
    let cardFaceTemplate;
    verid.load(apiSecret).then(function() {
      console.log('Ver-ID loaded');
      // Ver-ID loaded
      // Collect an ID card
      let front = new veridcredentials.Page(veridcredentials.ISOFormat.id1, new veridcredentials.FacePhotoFeature());
      let back = new veridcredentials.Page(veridcredentials.ISOFormat.id1, new veridcredentials.BarcodeFeature());
      // Specify the ID document
      let document = new veridcredentials.IDDocument([front, back]);
      // Create settings
      var settings = new veridcredentials.IDCaptureSessionSettings(document);
      // To detect a face you will be using for face recognition:
      settings.detectFaceForRecognition = true;
      // To show a session guide to the user:
      settings.showGuide = true;
      // To show the result of the session to the user:
      settings.showResult = true;
      return veridcredentials.captureId(settings);
    }).then(function(response) {
      console.log('Ver-ID response:',response);
      cardFaceTemplate = response.document.pages[0].faces[0].faceTemplate;
      // Capture live face
      var settings = new verid.LivenessDetectionSessionSettings();
      settings.includeFaceTemplateInResult = true;
      return verid.captureLiveFace(settings);
    }).then(function(result) {
      console.log('Ver-ID result:',result);
      if (result.outcome == verid.SessionOutcome.SUCCESS) {
        var liveFaceTemplate = result.getFaces(verid.Bearing.STRAIGHT)[0].faceTemplate;
        return verid.compareFaceTemplates(cardFaceTemplate, liveFaceTemplate);
      } else if (result.outcome != verid.SessionOutcome.CANCEL) {
        throw new Error(result.outcome);
      }
    }).then(function(score) {
      console.log('Ver-ID score:',score);
      if (score) {
        // Score is a value between 0.0 and 1.0 indicating similarity between the face templates. 
        // Higher scores mean higher similarity.
      } else {
        // Session canceled
      }
    }).catch(function(error) {
      console.log('Ver-ID error:',error);
      // Handle the failure
    });
```
xcode log:
```
2019-02-07 18:28:30.901402+0600 demoapp[5926:3058189] Ver-ID loaded
2019-02-07 18:28:30.946425+0600 demoapp[5926:3058189] Ver-ID error:
- 18:28:49.836 0x1c011d7f0 DEBUG: image already in database with hash: N/A and path: 
- 18:28:49.837 0x1c011d7f0 DEBUG: running full face detection with no limit on faces to find.
- 18:28:50.246 0x1c011d7f0 DEBUG: found 1 faces.
- 18:28:50.246 0x1c011d7f0 DEBUG: put face in background task with id: -3
- 18:28:50.757 0x1c011d7f0 DEBUG: Encrypting 680 bytes of data with key: No_encryption
- 18:28:50.778 0x1c011d7f0 DEBUG: saved face id: 2 with image width = 1086 and height = 663 and hash = bf1e6c82e70dda807bf2bc16c8a431e81d1d699a97a0f1c6454f63ed17838091
2019-02-07 18:28:50.780391+0600 demoapp[5926:3058320] Returning binary face template for face id: 2
2019-02-07 18:29:18.377240+0600 demoapp[5926:3058189] [Snapshotting] Snapshotting a view (0x107128c00, UIKeyboardImpl) that is not in a visible window requires afterScreenUpdates:YES.
```
I have attached screenshot below.
![img_0238](https://user-images.githubusercontent.com/6288562/52412150-1eeb9880-2b08-11e9-9564-ca1b075d5867.PNG)
And also I did not get any **response** after `veridcredentials.captureId(settings)` done.

**System info:**
```
xcode: Version 10.1 (10B61)
Device: iPhone 7 plus v 11.4
```
**Ionic info:**
```
   ionic (Ionic CLI)  : 4.0.1
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.1.8

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.4, browser 5.0.4, ios 4.5.4

System:

   Android SDK Tools : 26.1.1
   ios-deploy        : 1.9.4
   NodeJS            : v8.11.2 (/usr/local/bin/node)
   npm               : 6.2.0
   OS                : macOS High Sierra
   Xcode             : Xcode 10.1 Build version 10B61

Environment:

   ANDROID_HOME : /Users/bs228/Library/Android/sdk
```
I would greatly appreciate your help on this.
Thank you

**Comment by jakubdolejs** [comment]
Fixed in c7274319add783e414f268a59ba4a74e8f5e1686

**Comment by sanidkpc** [comment]
Hi @jakubdolejs,
Thanks for the commit [c727431](https://github.com/AppliedRecognition/Ver-ID-Credentials-Cordova-Plugin/commit/c7274319add783e414f268a59ba4a74e8f5e1686). But this fix only works on Android. I verify and test, I got the response and result for Android,
but did not got anything for **iOS**. 
No response or result showing on iOS
This issues is not resolve for iOS.
Thank you
looking forward to hearing from you
